frontend/src/metabase/core/components/Button/Button.styled.tsx

```diff
@@ -23,6 +24,21 @@ export const ButtonRoot = styled.button<ButtonRootProps>`
         border-color: ${alpha("filter", 0.88)};
       }
     `}
+
+  ${({ onlyText }) =>
+    onlyText &&
+    css`
+      border: none;
+      padding: 0;
+
+      ${ButtonContent} {
```

Comment:
Could we simply set `color` on the top level component without targeting `ButtonContent`? It seems that `purple` variant does that.

Reply:
Certainly can!